.gitlab-ci.yml

```diff
@@ -50,7 +43,7 @@ variables:
         --tag $REGISTRY_PATH/$IMAGE_NAME:$IMAGE_DATE_TAG
         --tag $REGISTRY_PATH/$IMAGE_NAME:$IMAGE_TAG
         --file $IMAGE_NAME/Dockerfile .
-    - docker login -u "${Docker_Hub_User_Parity}" -p "${Docker_Hub_Pass_Parity}"
+    - docker login -u "${DOCKER_USER}" -p "${DOCKER_PASSWORD}"
```

Comment:
what about using a syntx like here: https://docs.docker.com/engine/reference/commandline/login/ along with https://docs.gitlab.com/ee/ci/variables/#custom-environment-variables-of-type-file?
the password varable will not be masked in the scripts (would be nice though)